index.js

```diff
@@ -0,0 +1,28 @@
+import Books from './modules/Books.js';
+
+import { DateTime } from './modules/date.js';
+
+import {
+  even,
+  showhide,
+} from './modules/elements.js';
+
+Books.displayall();
+
+document.querySelector('.add').addEventListener('click', even);
+
+const timeDate = document.querySelector('.date');
+
+const dateAndTime = () => {
+  setInterval(() => {
+    const date = (`${DateTime.now().toLocaleString(DateTime.DATETIME_MED)}`);
+    timeDate.innerHTML = date;
+  }, 0);
+};
+
+dateAndTime();
```

Comment:
- [ ] Please make sure after making the changes, the app is still working and it is possible to add and remove books from the list. Currently when ever you try to click a  nav link it just changes the URL but nothing else changes



<video width="320" height="240" src="https://github.com/vikas924/awesomebookes6/assets/44798044/b80ea258-d152-4cfa-9249-772465c0bd5f"/>